frontend/src/views/Game.tsx

```diff
@@ -52,11 +65,28 @@ function GamePage() {
    */
   useBeforeunload(() => 'Leaving this page may cause you to lose your current code and data.');
 
+  const setStateFromGame = useCallback((game: Game) => {
+    setRoom(game.room);
+    setPlayers(game.players);
+    setGameTimer(game.gameTimer);
+    setProblems(game.problems);
+
+    // Print room and currentPlayer to prevent unused variable warning
+    console.log(room);
+    console.log(currentPlayer);
```

Comment:
I think this might actually create a circular dependency issue looking at the console haha (adding currentPlayer as a useCallback dependency causes the setCurrentPlayer method to trigger it over and over) - I fixed this in my next branch (which builds on this PR) since idk if it causes any noticeable UI bugs, but lmk and I can also implement that change here

Reply:
Actually I ended up removing it but I think there's something still going on - see comment chain below